sdk/cosmosdb/cosmos/src/utils/envUtils.ts

```diff
@@ -4,7 +4,5 @@
 import process from "node:process";
 import { Constants } from "../common/constants";
 
-export const disableListAndSetAggregate: boolean =
-  process.env.DISABLE_LIST_AND_SET_AGGREGATE === "true";
```

Comment:
What about "True", "1", "enabled", etc? Or do we document that only "true" is accepted?